cookiecutter/repository.py

```diff
@@ -3,13 +3,15 @@
 import re
 
 from cookiecutter.exceptions import RepositoryNotFound
-from cookiecutter.vcs import clone
+from cookiecutter.vcs import clone, REPO_TYPES
 from cookiecutter.zipfile import unzip
 
 REPO_REGEX = re.compile(
     r"""
 # something like git:// ssh:// file:// etc.
-((((git|hg)\+)?(git|ssh|file|https?):(//)?)
+(((("""
+    + '|'.join(REPO_TYPES.keys())
```

Comment:
Hey @Theta-Dev, with this type of expansion, you are catching correctly `svn+<schema>` like :

```
svn+https://private.com/gitrepo
```

but not:

```
svn://private.com/gitrepo
```

you should consider to add `svn` also in the second regex group like `(git|svn|ssh|file|https?)`

and add some test case on 

https://github.com/cookiecutter/cookiecutter/blob/52dd18513bbab7f0fbfcb2938c9644d9092247cf/tests/repository/test_is_repo_url.py#L25-L38

Reply:
thx